packages/styles/scss/patterns/sub-patterns/link-list/_link-list.scss

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@mixin themed-items {
+  .#{$prefix}--link--list__heading {
+    color: $text-01;
+  }
+}
+@mixin link--list {
+  .#{$prefix}--link--list {
```

Comment:
Wondering if the class name should be `.bx--link-list`?

Reply:
I think we used {prefix} for `bx` place @jeffchew in other scss too.

Reply:
What I meant was why is it `link--list` and not `link-list` (single dash)?

Reply:
Okay got it